packages/wonder-blocks-popover/src/components/__docs__/popover-content.argtypes.js

```diff
@@ -0,0 +1,116 @@
+    singleButtonEmphasized: (
+        <Button kind="primary" light={true}>
+            Continue
+        </Button>
+    ),
+    pagination: (
+        <>
+            <Button kind="tertiary">Previous</Button>
+            <Strut size={Spacing.medium_16} />
+            <Button kind="primary">Next</Button>
+        </>
+    ),
+    paginationEmphasized: (
+        <>
+            <Button kind="tertiary" light={true}>
+                Previous
+            </Button>
+            <Strut size={Spacing.medium_16} />
+            <Button kind="primary" light={true}>
+                Next
+            </Button>
+        </>
+    ),
+    steps: (
+        <>
+            <LabelLarge>Step 1 of 5</LabelLarge>
+            <Spring />
+            <Button kind="tertiary">Skip this step</Button>
+        </>
+    ),
+    stepsEmphasized: (
+        <>
+            <LabelLarge>Step 1 of 5</LabelLarge>
+            <Spring />
+            <Button kind="secondary" light={true}>
+                Skip this step
+            </Button>
+        </>
+    ),
+};
+
+export default {
+    content: {
+        description: "The content to render inside the popover.",
+        type: {required: true, summary: "string"},
+    },
+    title: {
+        description: "The popover title",
+        type: {required: true, summary: "string"},
+    },
+    actions: {
+        description: `User-defined actions.\n\nIt can be either a Node or a function using the children-as-function pattern to pass a close function for use anywhere within the actions. This provides a lot of flexibility in terms of what actions may trigger the Popover to close the popover dialog`,
+        type: {
+            summary: "React.Node | (({|close: () => mixed|}) => React.Node)",
+        },
+        options: (Object.keys(ActionsMappings): Array<React.Node>),
+        mapping: ActionsMappings,
+    },
+    closeButtonVisible: {
+        description: `When true, the close button is shown; otherwise, the close button is not shown.`,
+        table: {
+            type: {
+                summary: "boolean",
+            },
+        },
+    },
+    closeButtonLabel: {
+        description: "Close button label for use in screen readers",
+        control: {type: "text"},
+    },
+    style: {
+        description: "Optional custom styles.",
+        control: {type: "object"},
+        table: {
+            category: "Styling",
+            type: {
+                summary: "StyleType",
+            },
+        },
+    },
+    testId: {
+        description: "Test ID used for e2e testing.",
+        control: {type: "text"},
+        table: {
+            type: {
+                summary: "string",
+            },
+        },
+    },
+    icon: {
+        description: `Decorate the popover with an illustrated icon. It cannot be used at the same time with image.`,
+        type: {
+            summary: `string | React.Element<"img"> | React.Element<"svg">`,
+        },
+    },
+    image: {
```

Comment:
I'm also getting an error if I try to set the object for the `image` prop.

Reply:
Will do the same here.